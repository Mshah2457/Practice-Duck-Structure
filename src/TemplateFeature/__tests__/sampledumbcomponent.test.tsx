import * as React from 'react';
import SampleDumbComponent from '../components/sampleDumbComponent';

import * as renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const rendered = renderer.create(<SampleDumbComponent foo={"bar"} bar={"baz"} />).toJSON();
  expect(rendered).toBeTruthy();
});

test('matches snapshot', () => {
  const rendered = renderer.create(<SampleDumbComponent foo={"bar"} bar={"baz"} />).toJSON();
  expect(rendered).toMatchSnapshot();
});
