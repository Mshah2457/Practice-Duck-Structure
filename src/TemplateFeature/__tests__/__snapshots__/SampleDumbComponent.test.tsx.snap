// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    FOO: bar
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    BAR: baz
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    BAZ: baz
  </Text>
</View>
`;
